.github/workflows/ci.yml

```
@@ -1,20 +1,243 @@
+              ('export const fetchFxStatus = Effect.fn("FxTwitter.fetchStatus")(function* (', 'export const fetchFxStatus = Effect.fn("FxTwitter.fetchStatus")(function* fetchFxStatusEffect('),
+              ('export const fetchFxThread = Effect.fn("FxTwitter.fetchThread")(function* (', 'export const fetchFxThread = Effect.fn("FxTwitter.fetchThread")(function* fetchFxThreadEffect('),
+              (')(function* (id: string, rankingMode: FxReplyRanking = "likes", limit = 10) {', ')(function* fetchFxConversationRepliesEffect(id: string, rankingMode: FxReplyRanking = "likes", limit = 10) {'),
+              ('export const fetchFxFullThread = Effect.fn("FxTwitter.fetchFullThread")(\n  function* (id: string) {', 'export const fetchFxFullThread = Effect.fn("FxTwitter.fetchFullThread")(\n  function* fetchFxFullThreadEffect(id: string) {'),
+          ]
+          for old, new in replacements:
+              if old not in text:
+                  raise SystemExit(f"missing fxtwitter replacement: {old[:100]!r}")
+              text = text.replace(old, new, 1)
+
+          old = '''      replying_to_status:
+        raw.replying_to_status === null
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)